src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -373,7 +388,7 @@ public boolean isCourseEditFormEnabled() {
     }
     
     public void changeCourseIdInForm(int instrNum, String newCourseId) {
-        String selector = "$('#edit-" + instrNum + "').find('[name=\"courseid\"]')";
+        String selector = "$('#edit-" + instrNum + "').find('[name=\"" + Const.ParamsNames.COURSE_ID + "\"]')";
```

Comment:
Is it possible to use a child selector here?
Like `"$('#edit-" + instrNum + " > input[name=\'" + Const.ParamsNames.COURSE_ID + "\']')"`

// still too messy though =(